x-pack/plugins/observability/public/components/shared/core_web_vitals/index.tsx

```diff
@@ -38,6 +38,20 @@ export function formatToSec(value?: number | string, fromUnit = 'MicroSec'): str
   return (valueInMs / 1000).toFixed(2) + ' s';
 }
 
+function formatToMilliseconds(value?: number | null) {
+  if (typeof value === 'undefined' || value === null) {
+    return null;
+  }
+  return formatToSec(value, 'ms');
+}
+
+function formatToFixed3(value?: number | null) {
+  if (typeof value === 'undefined' || value === null || typeof value !== 'number') {
```

Comment:
@andrewvc You mentioned I shouldn't need the check against number because of the runtime checks from `io-ts`, is that correct?

Reply:
Right, it's guaranteed to be either a `null | number`, I'd say we don't even need this function given that simplicity. We can just call `toFixed(3)` inline where needed, and use `?` to handle the null case.